### PR TITLE
[example]Fuzz With Session

### PR DESCRIPTION
**Pull request by dark-lbp**
This fuzzer performs server fuzzing with session enabled.

### CODE REVIEW

File: examples/05_fuzz_with_session/session_server.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    The SessionHandler class for our server.
+    First you need request a session with op_code(1), Server will return a specific session for this connect.
+    After you got the session you can use op_code(2) with correct session to send data to Server.
+    When Server recived op_code(2) with correct session, it will send back the packet which you send to Server.
+
+    :Example Protocol:
+
+        ::
+            Fuzzer (client)                               Target (server)
+                ||---------------(get_session)----------------->||
```
Comment (BinyaminSharet) [inline examples/05_fuzz_with_session/session_server.py]: Align this line (add one '-')

File: examples/05_fuzz_with_session/runner.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
+# Copyright (C) 2016 Cisco Systems, Inc. and/or its affiliates. All rights reserved.
+#
+# This file is part of Kitty.
+#
```
Comment (BinyaminSharet) [inline examples/05_fuzz_with_session/runner.py]: Add yourself as author (as in the katnip stuff)

File: examples/05_fuzz_with_session/runner.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    UInt8(value=2, name='op_code', fuzzable=False),
+    Dynamic(key='session_id', default_value='\x00\x00'),
+    String(name='data', value='some data')
+])
+
+
+def new_session_callback(fuzzer, edge, resp):
+    '''
+    :param fuzzer: the fuzzer object
+    :param edge: the edge in the graph we currently at.
```
Comment (BinyaminSharet) [inline examples/05_fuzz_with_session/runner.py]: Fix doc string. It seems like you kept the one from my example.

File: examples/05_fuzz_with_session/runner.py [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+                 edge.src is the open_session template
+                 edge.dst is the do_something template
+    :param resp: the response from the target
+    '''
+    # let's say that we have a reference to the target object,
```
Comment (BinyaminSharet) [inline examples/05_fuzz_with_session/runner.py]: This comment is irrelevant

File: examples/05_fuzz_with_session/README.rst [hunk 5]
```diff
@@ -0,0 +1,11 @@
+Server Fuzzer - Fuzz With Session
+=================================
+
+
+This fuzzer performs server fuzzing with session enabled.
+
```
Comment (BinyaminSharet) [inline examples/05_fuzz_with_session/README.rst]: Add some description here

**Comment by dark-lbp** [comment]
This is Example Report looks like.
example-crash-report

**Pull request merged**